### ISSUE TITLE
Full screen mode (hide status bar)

### ISSUE THREAD

**Issue by Sid8161** [kiwibrowser/src.next#160]
Hi guys, is there a chance to add an option "Full Screen Mode" (immersive) to hide status bar, like it is on Via browser ?

**Comment by allytiago** [comment]
It's already the end of 2022 and I still haven't seen the option

**Comment by Ibuprophen** [comment]
> It's already the end of 2022 and I still haven't seen the option

There is a Chrome extension called 'Shortkeys' that may be what you're looking for. It lets you, among other things, map custom JavaScript snippets to keyboard shortcuts. It looks like you need to pay for it if you download it from the chrome web store.

The following link is to the Github Project for the 'Shortkeys' extension. 

https://github.com/crittermike/shortkeys

https://github.com/crittermike/shortkeys/releases

Basically, it allows you to write custom JavaScript shortcuts and lets you assign it to a keyboard shortcut/combinations to use.

It does other things too but this ability to run custom JavaScript is what your looking for. 

Good Luck! :-)) 

~Ibuprophen

**Comment by KhalfaniW** [comment]
`document.documentElement.webkitRequestFullScreen();`
Does not so the problem because it's visible in fullscreen. 

You can test it without extensions by pasting it in the developer console

**Comment by allytiago** [comment]
It would be useful if there was a shortcut for such a command like F11

Em ter., 6 de jun. de 2023 22:01, KhalfaniW ***@***.***>
escreveu:

**Comment by thsea** [comment]
the three years have passed,...I really need it, please

**Comment by vanarok** [comment]
![Screenshot_2023-10-30-10-09-44-751_com vivaldi browser](https://github.com/kiwibrowser/src.next/assets/6105977/d63073cd-4550-4fbd-a702-dbf53f524831)
![Screenshot_2023-10-30-10-12-12-757_com vivaldi browser](https://github.com/kiwibrowser/src.next/assets/6105977/8dabb1b4-e704-47d6-a4d3-cbbc793f04b2)
This small but nice feature makes using web applications more enjoyable. For example, I like coding in vscode web, and having the status bar disabled would not distract me from development (notifications, time, etc.) and would provide extra space for the application. Please implement this feature.
